callbacks/image_viewer.py

```diff
@@ -25,13 +15,46 @@
     downscale_view,
     get_view_finder_max_min,
     resize_canvas,
+    resize_canvas_with_zoom,
 )
 
-clientside_callback(
-    ClientsideFunction(namespace="clientside", function_name="get_container_size"),
-    Output("screen-size", "data"),
-    Input("url", "href"),
+
+@callback(
+    Output("image-viewer", "figure", allow_duplicate=True),
+    Output("annotation-store", "data", allow_duplicate=True),
+    Output("image-viewfinder", "figure", allow_duplicate=True),
+    Input("window-resize", "width"),
+    Input("window-resize", "height"),
+    State("annotation-store", "data"),
+    prevent_initial_call=True,
 )
+def resize_window_event(screen_width, screen_height, annotation_store):
```

Comment:
Is it possible to switch this to a clientside callback for better performance?

Reply:
I don't see why not